demos/notebooks/demo_pipeline_cnmfE.ipynb

```diff
@@ -51,6 +64,7 @@
     "    pass\n",
     "import bokeh.plotting as bpl\n",
```

Comment:
Ordinarily I'd suggest putting these up with the other pre-caiman imports, but I'm guessing this needs to be here so it happens after the run_line_magic?

Reply:
Actually I'm not sure why those are like that.